service/frontend/workflowHandler.go

```diff
@@ -1349,11 +1349,21 @@ func (wh *WorkflowHandler) StartWorkflowExecution(
 			Message: "A valid ExecutionStartToCloseTimeoutSeconds is not set on request."}, scope)
 	}
 
+	if startRequest.GetExecutionStartToCloseTimeoutSeconds() > common.MaxWorkflowTimeout {
```

Comment:
I think we decided to not restrict workflow timeout and instead have a max limit on decision task timeout of 1 year.

Reply:
Updated.